recipes/machina/meta.yaml

```diff
@@ -14,30 +14,35 @@ source:
     folder: gurobi902
 
 build:
-  number: 0
-  skip: True  # [osx]
+  number: 1
+  skip: True  # [osx or py < 38]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - cmake>=3.9
   host:
+    - python
```

Comment:
Does this just need python for building? If so, you can put it in the build section.